### PR TITLE
Treewide: Add altitude attribute

### PR DESCRIPTION
**Pull request by FFHener**
This will help with further automating the documentation like wiki, locationpages at our website or uisp.

- [x] rename height to altitude
- [x] Explain in README [altitude vs height](https://en.wikipedia.org/wiki/Altitude#/media/File:Vertical_distances.svg)

### CODE REVIEW

**Comment by FFHener** [comment]
i will fix the typo, 1 mom

File: locations/dragonkiez-adlerhalle.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ location: dragonkiez-adlerhalle
 location_nice: Dragonkiez Adlerhalle
 latitude: 52.495065429
 longitude: 13.387666941
+altitude: 2
```
Comment (Noki) [inline locations/dragonkiez-adlerhalle.yml:6]: can't be right

File: locations/dragonkiez-buero.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ location: dragonkiez-buero
 location_nice: Dragonkiezbuero
 latitude: 52.494722496
 longitude: 13.387774229
+altitude: 2
```
Comment (Noki) [inline locations/dragonkiez-buero.yml:6]: can't be right

File: locations/dragonkiez-dorfplatz.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ location: dragonkiez-dorfplatz
 location_nice: Dragonkiezdorfplatz
 latitude: 52.495457349
 longitude: 13.386079073
+altitude: 2
```
Comment (Noki) [inline locations/dragonkiez-dorfplatz.yml:6]: can't be right

File: locations/dragonkiez-kiezraum.yml [hunk 4]
```diff
@@ -3,6 +3,7 @@ location: dragonkiez-kiezraum
 location_nice: Dragonkiez Kiezraum
 latitude: 52.494807413
 longitude: 13.387511373
+altitude: 2
```
Comment (Noki) [inline locations/dragonkiez-kiezraum.yml:6]: can't be right

File: locations/dragonkiez-plangarage.yml [hunk 5]
```diff
@@ -3,6 +3,7 @@ location: dragonkiez-plangarage
 location_nice: Dragonkiez Plangarage
 latitude: 52.49550240409573
 longitude: 13.38777191534464
+altitude: 2
```
Comment (Noki) [inline locations/dragonkiez-plangarage.yml:6]: can't be right

File: locations/dragonkiez-rathausblock-miami.yml [hunk 6]
```diff
@@ -3,6 +3,7 @@ location: dragonkiez-rathausblock-miami
 location_nice: Dragonkiez Rathausblock Miami
 latitude: 52.495855798
 longitude: 13.387956619
+altitude: 2
```
Comment (Noki) [inline locations/dragonkiez-rathausblock-miami.yml:6]: can't be right

File: locations/k12.yml [hunk 7]
```diff
@@ -3,6 +3,7 @@ location: k12
 location_nice: 'Kastanienallee 12, Haus 2'
 latitude: 52.53936534993554
 longitude: 13.409738833169316
+altitude: 105
```
Comment (Noki) [inline locations/k12.yml:6]: can't be right

**Review by Noki — CHANGES_REQUESTED** [review]
some altitudes can't be right

File: locations/dragonkiez-adlerhalle.yml (hunk 1 above)
Comment (FFHener) [reply locations/dragonkiez-adlerhalle.yml:6]: This value is from the wiki. Its likely the height. According to google earth the altitude could be 31. Should we go with this value and set the `2` as a `height`?

File: locations/k12.yml (hunk 7 above)
Comment (Noki) [reply locations/k12.yml:6]: about 62-64m

File: locations/dragonkiez-adlerhalle.yml (hunk 1 above)
Comment (Noki) [reply locations/dragonkiez-adlerhalle.yml:6]: yes, please use 2m as a height and elevation here is 33m so altitude would be 35m

File: locations/dragonkiez-buero.yml (hunk 2 above)
Comment (Noki) [reply locations/dragonkiez-buero.yml:6]: please use 2m as a height and elevation here is 33m so altitude would be 35m

File: locations/dragonkiez-dorfplatz.yml (hunk 3 above)
Comment (Noki) [reply locations/dragonkiez-dorfplatz.yml:6]: please use 2m as a height and elevation here is 33m so altitude would be 35m

File: locations/dragonkiez-kiezraum.yml (hunk 4 above)
Comment (Noki) [reply locations/dragonkiez-kiezraum.yml:6]: please use 2m as a height and elevation here is 33m so altitude would be 35m

File: locations/dragonkiez-plangarage.yml (hunk 5 above)
Comment (Noki) [reply locations/dragonkiez-plangarage.yml:6]: please use 2m as a height and elevation here is 35m so altitude would be 37m

**Comment by FFHener** [comment]
> some altitudes can't be right

I edited the proposed data

**Review by Noki — APPROVED** [review]
looks all plausible to me now

**Pull request merged**